docs/UserGuide.md

```diff
@@ -330,12 +330,16 @@ Now, your data should be successfully transferred to the new computer.
-**Help** | `help`                                                                                                                                                                                                                                          |
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find alex david`                                                                                                                                                                                      |
+| Action         | Format, Examples                                                                                                                                                                                                                                |
+|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**        | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS s/SALARY l/LEAVE r/ROLE d/DEPARTMENT… m/MANAGER NAME…` <br> e.g., `add n/Johnny p/91242712 e/johnnysins@gmail.com a/Johnny street, block 69, #05-05 s/5300 l/14 r/subordinate d/ R&D m/ Alex Yeoh` |
+| **Clear**      | `clear`                                                                                                                                                                                                                                         |
+| **Delete**     | `delete INDEX`<br> e.g., `delete 4`                                                                                                                                                                                                             |
+| **Department** | `delete t/(add/delete> n/DEPARTMENT_NAME` <br> e.g., `department t/add n/Engineering`                                                                                                                                                           |
+| **Edit**       | `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [s/SALARY] [l/LEAVE] [r/ROLE] [m/MANAGER NAME]… [d/DEPARTMENT]…`<br> e.g.,`edit 1 p/91234567 e/johnsimmons@gmail.com`                                                                      |
+| **Exit**       | `exit`                                                                                                                                                                                                                                          |
+| **Filter**     | `filter DEPARTMENT_NAME [more DEPARTMENT_NAME]` <br> e.g., `filter R&D`                                                                                                                                                                                     |
```

Comment:
could we change the `filter` command summary to:

`filter [n/NAME] [e/EMAIL] [a/ADDRESS] [s/SALARY] [l/LEAVE] [r/ROLE] [m/MANAGERNAME] [d/DEPARTMENT]` <br> e.g. `filter d/R&D`

because it uses prefixes now